File: data/duoglot/tests/basic/js/M_temp.js
let bar = true;
let y = [1, [21, 22], 3, 4, 5];
let str_list = [];
for (let x of y) {
  str_list.push(x.toString());
}
let c = " good nice";
console.log("hi" + bar.toString());

let a = 0;
for (let i = 0; i < 10; i++) {
  for (let j = 10; j > i; j -= 1) {
    a += Math.abs(i * j);
  }
}

function baz(x, y, z) {}

function foo(x, y) {
  a = x;
  let z = x + y;
  if (bar) {
    baz("hello", x, y);
    return z;
  }
  return x - y;
}

try {
  throw Error("CustomError");
} catch (e) {
  console.log("Exception!");
  console.log(e);
} 

File: data/duoglot/tests/basic/py/M_temp.py
bar = True
y = [1, [21, 22], 3, 4, 5]
str_list = [str(x) for x in y]
c = " good nice";
print("hi" + str(bar))

a = 0
for i in range(10):
  for j in range(10, i, -1):
    a += abs(i * j)

def baz(x, y, z): pass

def foo(x, y):
  a = x
  z = x + y
  if bar:
    baz("hello", x, y)
    return z
  return x - y

try:
  raise Exception("CustomError")
except Exception as e:
  print("Exception!")
  print(e)